control_plane/src/bin/neon_local.rs

```diff
@@ -1202,15 +1220,18 @@ async fn handle_safekeeper(sub_match: &ArgMatches, env: &local_env::LocalEnv) ->
     Ok(())
 }
 
-async fn handle_start_all(env: &local_env::LocalEnv) -> anyhow::Result<()> {
+async fn handle_start_all(
+    env: &local_env::LocalEnv,
+    retry_timeout_in_seconds: u64,
```

Comment:
Why not  `std::time::Duration`?

With that, you can shorted the variable name to `retry_timeout` (please propagate that change down the call graph) and there's more type safety.

Reply:
good point, the const variable used before for the retry timeout was a u64 and I "sticked to the convention" but you are right